### PR TITLE
Feat: Added Modal and Rating

### PR DESCRIPTION
**Pull request by sainath7878**
Added Modal and Rating in Documentation site.

Modal:
Screenshot 2022-02-20 at 5 01 34 PM

Rating:
Screenshot 2022-02-20 at 5 02 15 PM

### Thanks for reviewing

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -994,8 +998,127 @@ <h2 class="fs-m">JavaScript</h2>
...
+                    const showModal = document.querySelector(".btn-show-modal");
+
+                    showModal.addEventListener("click",() => {
+                        modalContainer.style.display = "block";
+                    })
+                    
+                    modalClose.addEventListener("click",()=>{
+                        console.log("clicked");
+                        modalContainer.style.display = "none";
+                    })
```
Comment (vishal804) [inline docs.html:1064]: I guess  are having same script code in index.js file 
do check it once

File: component/index.js [hunk 2]
```diff
@@ -25,4 +32,15 @@ const showSnackbar = () => {
...
+snackbarHideBtn.addEventListener("click", hideSnackbar);
+
+
+// ------ script for Modal --------- 
+showModal.addEventListener("click",() => {
+    modalContainer.style.display = "block";
+})
+
+modalClose.addEventListener("click",()=>{
+    console.log("clicked");
```
Comment (vishal804) [inline component/index.js]: I guess this console is for testing purpose 
so  can remove this once it is done

Comment (sainath7878) [reply component/index.js]: Yes, I forgot to remove. Thanks for pointing out.

File: docs.html [hunk 3]
```diff
@@ -994,8 +998,127 @@ <h2 class="fs-m">JavaScript</h2>
 
                 </script>
```
Comment (05him) [inline docs.html:999]: suggestion:- useless spaces here

**Review by 05him — APPROVED** [review]
The code and UI Looks good to me , 
suggestion :- there are few extra useless spaces  that can be removed

**Review by vishal804 — APPROVED** [review]

**Review by abhisheksahani-nova — APPROVED** [review]
Looks good to go , just make changes as suggested by vishal

File: docs.html (hunk 1 above)
Comment (sainath7878) [reply docs.html:1064]: This is for modal demo which is not included in components

**Review by BalajiHarinath — APPROVED** [review]
The code looks good you can merge.

File: docs.html (hunk 3 above)
Comment (sainath7878) [reply docs.html:999]: I gave them to distinguish between code snippets and the actual actual

**Pull request merged**